frontend/src/static_pages.tsx

```diff
@@ -202,20 +202,20 @@ export function Kontakt() {
 
 export function Footer() {
   return (
-    <div className="footer">
+    <Paper className="footer">
```

Comment:
I wonder why you made the footer a `Paper`. Was there a specific reason? I think, now it doesn't look like a real footer any more - it doesn't stick to the bottom of the page and has space around it on the left and right. 
Not sure if we should focus on this now but I think, at least at some point, it should be improved.

Reply:
the paper component just made round borders, basically. I moved the footer up, but maybe was better as before, I will leave it more at the bottom. The spaces where there before. We might do some redesign later.